src/spotted/config/db/post_db_init.sql

```diff
@@ -5,6 +5,7 @@ CREATE TABLE IF NOT EXISTS pending_post
   u_message_id BIGINT NOT NULL,
   g_message_id BIGINT NOT NULL,
   admin_group_id BIGINT NOT NULL,
+  credit_username VARCHAR(255) DEFAULT NULL,
```

Comment:
one day we should implement a pending_update release sql files for this kind of updates